superset/assets/javascripts/explore/stores/controls.jsx

```diff
@@ -955,6 +955,18 @@ export const controls = {
     description: 'Whether to include a time filter',
   },
 
+  filter_order: {
+    type: 'SelectControl',
+    multi: true,
+    label: 'Order by optional value',
+    default: [],
+    mapStateToProps: state => ({
+      choices: (state.datasource) ? state.datasource.order_by_choices : [],
```

Comment:
If I want to compare the options ordering by`metric descending` or `metric ascending`, I  would prefer to create a new slice specially to show that.